docs/business-apps/power-automate/get-started/create-your first-flow.md

```diff
@@ -49,16 +46,16 @@ Before proceeding with these instructions, make sure your SharePoint site is set
 
     ![Flow template - send email when an item is added in a list](../../../images/gs01-create-a-flow-when-item-is-added-template.png)
 
-6. To create the flow, select **Create Flow**. Any connections that require a connection is also attempted during this step. After the flow is created successfully, you are redirected to the flow designer where you can edit and modifify the flow if needed.
+6. To create the flow, select **Create Flow**. Any connections that require a connection is also attempted during this step. After the flow is created successfully, you are redirected to the flow details page where you can edit and modifify the flow if needed.
+    >Your flow is active as soon as you save the flow. In this case, this flow that will send an email when new items are added to the list is now active, and runs whenever new items are added to the **Projects** list in the specified SharePoint site.
 
-    ![Flow designer - send email when an item is added in a list](../../../images/gs01-designer-when-item-is-added-template.png)
+    ![Flow designer - send email when an item is added in a list](../../../images/gs01-when-item-is-added-template-flow-created.png)
 
-7. Select the name of the flow that reads **Send a customized email when a new SharePoint list item is added**, and enter the following name for the flow:
-    * Send an email for new projects
+7. Click **Edit** in the top command bar to edit the flow in the flow designer.
 
-   The SharePoint actions are represented with the SharePoint logo in the flow designer.
+    ![Flow designer - send email when an item is added in a list](../../../images/gs01-designer-when-item-is-added-template.png)
 
-8. In the **When the new item is created** action, to expand the action, select **Edit**. Note the input configured to the SharePoint site address and list name from where you created the flow.
+8. Click on **When the new item is created** trigger to expand and view the configured properties.Note the input configured to the SharePoint site address and list name from where you created the flow.
```

Comment:
To expand and view the configured properties, select the **When the new item is created** trigger. Note the input configured to the SharePoint site address and list name from where you created the flow.